scripts/create_addon

```diff
@@ -33,6 +33,16 @@ DESCRIPTION
        --show-only
               output the list of packages, which are intented to build
 
+       --write-logs=[yes,no,errors]
+              write a log file per addon
+              yes - write for every addon a log (and keep it)
+              no - write no logs (default)
+              errors - only keep logs foir failed addons
```

Comment:
s/foir/for

Reply:
@hudokkow already fixed at https://github.com/LibreELEC/LibreELEC.tv/pull/2773/commits/b0012dccf58075644ec44d3701e53d33c9697882

Reply:
oops, missed that commit. sorry ;)